pkg/accountGRPC/account.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package accountGRPC;
+option go_package = "github.com/stasBigunenko/monorepa/pkg/account/proto";
+
+import "google/protobuf/empty.proto";
+
+service UserGRPCService {
+  rpc Get(User_id) returns (Accounts) {}
+  rpc Create(User_id) returns (Obj) {}
+  rpc Update(Balance) returns (Balance) {}
+  rpc Delete(User_id) returns (google.protobuf.Empty) {}
+}
```

Comment:
Add RPC for ListAccounts to display all accounts of all users.

Reply:
done